core/replay/connections_parser.py

```diff
@@ -89,6 +91,64 @@ def parse_connections(
     return connections, total_connections
 
 
+class Log:
```

Comment:
I believe this is to do with the StartNodeLog, if so, can you rename it to `StartNodeLog` ?